### PR TITLE
feat(examples): Add RAGAS evaluation to RAG chain

### PR DESCRIPTION
**Pull request by caetano-colin**
This PR add RAGAS evaluation to the GenAI Multimodal example.

It also adds steps on the README that instruct the user how to deploy using the infra pipeline (cloud build).

### CODE REVIEW

File: examples/genai-rag-multimodal/README.md [hunk 1]
```diff
@@ -59,6 +59,108 @@ When running the Notebook, you will reach a step that downloads an example PDF f
       - projects/200612033880 # Google Cloud Example Project
```
Comment (renato-rudnicki) [inline examples/genai-rag-multimodal/README.md:59]: Maybe we could change the `200612033880` for `project_number` or what that number means, or even `projects/00000000000`

**Review by renato-rudnicki — COMMENTED** [review]
Just a small comment, but LGTM.

File: examples/genai-rag-multimodal/README.md (hunk 1 above)
Comment (caetano-colin) [reply examples/genai-rag-multimodal/README.md:59]: In this case this project `200612033880` is a Google Cloud Projects that hosts a Storage Bucket that contains the PDF we are downloading

**Comment by caetano-colin** [comment]
Update: Release 5.34.0 is out and was fixed on versions.tf: https://github.com/hashicorp/terraform-provider-google/releases/tag/v5.34.0

**Comment by caetano-colin** [comment]
I was able to test the new provider version, the terraform version (5.34.0) fixes the issue we were having, this PR is ready to review/merge

**Pull request merged**